bsg_noc/bsg_wormhole_router.vh

```diff
@@ -9,3 +9,14 @@ typedef struct packed {                 \
   logic [in_len_width-1:0]    len;      \
   logic [in_cord_width-1:0 ]  cord;     \
 } in_struct_name
+
+`define declare_bsg_wormhole_router_packet_s(in_cord_width,in_len_width,in_payload_width,in_struct_name) \
```

Comment:
macro params are supposed to have "_mp" as suffix.

Reply:
yeah, you're right.  I was trying to be consistent with the rest of file.  I'll change both